gramps/gen/utils/docgen/odstab.py

```diff
@@ -335,7 +335,8 @@ def open(self,filename):
 
         try:
             self.content_xml = tempfile.mktemp()
-            self.f = open(self.content_xml,"wb")
+            with open(self.content_xml,"wb") as self.f:
```

Comment:
Looking back on the notes, `odstab.py` was identified as one of the files I should leave alone, so I'll update the PR and remove it.